recipe/meta.yaml

```diff
@@ -31,6 +31,7 @@ requirements:
     - libuuid
     - libxml2
     - krb5
+    - openssl =1
```

Comment:
```suggestion
    - openssl
```

Reply:
I don't get how this works. If we specify `openssl` without a version pin, how is it guaranteed that the version used at build time is the same as the version used at runtime? Even more so if we don't specify `openssl` as a run-time dependency at all.

From what I am reading on [`conda`'s packaging documentation](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirement) we could use `build.run_exports`,